content/en/faq/security/automtls-exlucde-port.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Does Auto mTLS work with the "excludeInboundPorts" annotations?
```

Comment:
This title is confusing in my opinion. What does "work with" mean?

I suggest something along the lines of... Does Auto mutual TLS exclude ports set using "excludeInboundPorts" annotation?

Reply:
Please also rename the file from `automtls-exlucde-port.md` to fix the typo.

Reply:
done.